libs/core/src/components/sage-tab/test/sage-tab.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { SageTab } from '../sage-tab';
+
+describe('sage-tabs', () => {
+  it('renders', async () => {
+    const page = await newSpecPage({
+      components: [SageTab],
+      html: `<sage-tab></sage-tab>`,
+    });
+    expect(page.root).toEqualHtml(`
+      <sage-tab slot="tabs">
+        <button aria-controls="undefined-panel" aria-selected="false" class="sage-tab" role="tab" tabindex="-1">
+          <div class="sage-tab__content"></div>
+        </button>
+      </sage-tab>
+    `);
+  });
+});
```

Comment:
We can add some more tests from a "unit" testing standpoint. Examples, setting props and what the expected output may be.